include/scrimmage/plugins/autonomy/ExternalControl/ExternalControl.h

```diff
@@ -60,14 +61,15 @@ class ExternalControl : public scrimmage::Autonomy {
  public:
     ExternalControl();
     virtual void init(std::map<std::string, std::string> &params);
-    bool step_autonomy(double t, double dt) final;
+    bool step_autonomy(double t, double dt);
```

Comment:
Is there a reason we need to take out the ``final``? The original design was to just have derived clases implement ``handle_action``, ``action_space_params``, ``calc_reward``, and if necessary ``close``.